components/common/CommonPaginator.vue

```diff
@@ -52,7 +52,7 @@ function percolateStatus(status) {
   addStatus(status)
 }
 
-nuxtApp.hook('elk-logo:click', () => {
+nuxtApp.hook('moz-logo:click', () => {
```

Comment:
This `elk-logo` -> `moz-logo` change isn't visible to the user, but it felt like a reasonable change to make for a tiny bit of developer experience improvement